src/ft2font.cpp

```diff
@@ -603,6 +603,34 @@ void FT2Font::load_char(long charcode, FT_Int32 flags, FT2Font *&ft_object, bool
     }
 }
 
+bool FT2Font::index_with_glyph(long charcode, int& index) const
```

Comment:
Not sure the name is clear. Also, do you need the `index`; can't it just return the `FT2Font`?

Reply:
Also, should `long` be `FT_ULong`?

Reply:
This is searching in a list of c++ objects that do not know about the Python wrappers, we need to return index to the next layer up so it can look up in the list of PyObjects which are the wrappers around the corresponding c++ objects.

Doing it this way means that we do not need to have a cycle between the PyObjects and c++ objects.

Reply:
Yes, I thought `fallbacks` was the same thing here and in the caller, but I see that it's just the same name but for lists of different objects.